scripts/gather_key_cyhy_metrics.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# A script to gather key Cyber Hygiene metrics from various sources.  Output
+# is a header row and a single line of comma-separated values.
+#
+# Usage:
+# gather_key_cyhy_metrics.sh cyhy_db_fqdn cyhy_reporter_fqdn cyhy_mongodb_uri \
+#   cyhy_mongodb_username cyhy_mongodb_password
+#
+# - cyhy_db_fqdn: The fully qualified domain name of the Cyber Hygiene database
+# server (e.g. "database.example.gov")
+# - cyhy_reporter_fqdn: The fully qualified domain name of the Cyber Hygiene
+# reporter server (e.g. "reporter.example.gov")
+# - cyhy_mongodb_uri: The MongoDB URI for the Cyber Hygiene database (e.g.
+# "mongodb://localhost:27017/cyhy" or "localhost/cyhy")
+# - cyhy_mongodb_username: The MongoDB username for the Cyber Hygiene database
+# - cyhy_mongodb_password: The MongoDB password for the Cyber Hygiene database
+#
+# Requires:
+# - SSH access to the Cyber Hygiene database and reporter servers
+# - mongosh installed
+# - A MongoDB user with read access to the Cyber Hygiene database
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ $# -eq 5 ]; then
+  cyhy_db_fqdn=$1
+  cyhy_reporter_fqdn=$2
+  cyhy_mongodb_uri=$3
+  cyhy_mongodb_username=$4
+  cyhy_mongodb_password=$5
+else
+  echo "Usage:  gather_key_cyhy_metrics.sh cyhy_db_fqdn cyhy_reporter_fqdn cyhy_mongodb_uri cyhy_mongodb_username cyhy_mongodb_password"
+  echo
+  echo "cyhy_db_fqdn: The fully qualified domain name of the Cyber Hygiene database server (e.g. \"database.example.gov\")"
+  echo "cyhy_reporter_fqdn: The fully qualified domain name of the Cyber Hygiene reporter server (e.g. \"reporter.example.gov\")"
+  echo "cyhy_mongodb_uri: The MongoDB URI for the Cyber Hygiene database (e.g. \"mongodb://localhost:27017/cyhy\" or \"localhost/cyhy\")"
+  echo "cyhy_mongodb_username: The MongoDB username for the Cyber Hygiene database"
+  echo "cyhy_mongodb_password: The MongoDB password for the Cyber Hygiene database"
+  exit 1
+fi
+
+today=$(date +%Y-%m-%d)
+
+# COMMANDER MAIN LOOP DURATION METRICS
+# NOTE: These metrics are pulled from all existing (non-compressed) commander
+# logs, not just the most recent one.
+commander_duration_mean=$(ssh "$cyhy_db_fqdn" "grep --only-matching --perl-regexp 'Last cycle took .* seconds' /var/log/cyhy/commander.log* | awk '{ total += \$4; count++ } END { print total/count }'")
```

Comment:
Just a thought to be specific since we're assuming the data is numeric.
```suggestion
commander_duration_mean=$(ssh "$cyhy_db_fqdn" "grep --only-matching --perl-regexp 'Last cycle took \d* seconds' /var/log/cyhy/commander.log* | awk '{ total += \$4; count++ } END { print total/count }'")
```

Reply:
The number we are capturing has a decimal and a single digit after the decimal in it, e.g.:
```
Last cycle took 1835.6 seconds.
```
 
Are you good with this instead?
```suggestion
commander_duration_mean=$(ssh "$cyhy_db_fqdn" "grep --only-matching --perl-regexp 'Last cycle took \d*.\d seconds' /var/log/cyhy/commander.log* | awk '{ total += \$4; count++ } END { print total/count }'")
```

Reply:
Yep that works for me. Sorry I forgot it's a float and not an int.

Reply:
This change was incorporated as part of https://github.com/cisagov/cyhy-system/pull/125/commits/803f7d0dbef2a5a29d0f57945bb7c6c610a5ddaf.